extensions/ohif-vtk-extension/package.json

```diff
@@ -91,6 +93,7 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.2.0",
+    "@ohif/i18n": "0.0.1",
```

Comment:
Should this be a dependency or peer dependency? If it's a dependency it will be duplicated at the app / extension level I think.

Reply:
Removed from vtk extension.
Its only needed in react-viewerbase package.json to run it lonelily.